src/components/TopBar/tests/TopBar.test.tsx

```diff
@@ -254,6 +244,13 @@ describe('<TopBar />', () => {
       expect(topBar).toContainReactComponent(TopBar.Menu);
     });
 
+    it('doesn’t render a logo when defined', () => {
+      const topBar = mountWithApp(
+        <TopBar contextControl={mockContextControl} />,
+      );
+      expect(topBar).not.toContainReactComponent(Image);
+    });
+
```

Comment:
I believe this is the test we wanted to colocate with the `describe('logo` block above.